database/engine/README.md

```diff
@@ -6,17 +6,88 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/database/engine/
 
 # Database engine
 
-The Database Engine works like a traditional database. It dedicates a certain amount of RAM to data caching and
+The Database Engine works like a traditional database. It dedicates a certain amount of RAM for data caching and
 indexing, while the rest of the data resides compressed on disk. Unlike other [database modes](/database/README.md), the
 amount of historical metrics stored is based on the amount of disk space you allocate and the effective compression
 ratio, not a fixed number of metrics collected.
 
 By using both RAM and disk space, the database engine allows for long-term storage of per-second metrics inside of the
 Agent itself.
 
-In addition, the dbengine is the only mode that supports changing the data collection update frequency
+In addition, the `dbengine` is the only mode that supports changing the data collection update frequency
 (`update every`) without losing the metrics your Agent already gathered and stored.
 
+## Tiering
+
+For Netdata Agent with version `netdata-1.35.0.138.nightly` and greater, `dbengine` supports tiering. Tiering is a
+mechanism of providing multiple tiers of data with
+different [granularity on metrics](/docs/store/distributed-data-architecture.md#granularity-of-metrics). Every Tier down
+samples the exact lower tier (lower tiers have greater resolution). You can have up to 5 Tiers **[0. . 4]** of data (
```

Comment:
downsamples is one word

Reply:
There's no "granularity of metrics" link on that document.

Reply:
it's not in this document, it's in `/docs/store/distributed-data-architecture.md#granularity-of-metrics`, resolving this comment